src/lib/configuration.ts

```diff
@@ -40,6 +40,7 @@ export class Configuration {
     notificationsUrl: '',
     investmentsUrl: '',
     username: undefined,
+    network: undefined,
```

Comment:
Is it right to declare `undefined` like this? Should it not be a machine readable / transferrable value like `null`?